resource/ui/ts/models/node_status.ts

```diff
@@ -141,10 +139,70 @@ module Models {
                         m("tr", [m("td", "Ranges:"), m("td", node.range_count)]),
                         m("tr", [m("td", "Leader Ranges:"), m("td", node.leader_range_count)]),
                         m("tr", [m("td", "Available Ranges:"), m("td", node.available_range_count)]),
-                        m("tr", [m("td", "Availablility:"), m("td", this._availability(nodeId))]),
+                        m("tr", [m("td", "Availablility:"), m("td", Nodes._availability(node))]),
```

Comment:
So this was interesting so I wanted to test it out.
When in a static method, the `this` refers to the class (or to be more exact, the function that is the definition of the class)
`
this
Stores()
`
referes to:
`
 var Stores = (function () {
            function Stores() {
                this._data = m.prop({});
                this.desc = m.prop({});
                this.statuses = m.prop({});
            }
            Stores.prototype.Query = function () {
  ...`